docs/algorithms/airl.rst

```diff
@@ -52,14 +53,22 @@ Detailed example notebook: :doc:`../tutorials/4_train_airl`
     rollouts = rollout.rollout(
         expert,
         env,
-        rollout.make_sample_until(min_timesteps=None, min_episodes=60),
-        rng=rng,
+        rollout.make_sample_until(min_episodes=60),
+        rng=np.random.default_rng(SEED),
     )
 
-    learner = PPO(env=env, policy=MlpPolicy)
+    env = make_vec_env("seals/CartPole-v0", rng=np.random.default_rng(SEED))
```

Comment:
Why do we re-create `env` rather than reusing the previous `env` object (perhaps reseeding as needed to make deterministic)?

Reply:
fixed